PendingReleaseNotes

```diff
@@ -1,5 +1,13 @@
 >=17.0.0
 
+* `ceph-mgr-modules-core` debian package does not recommend `ceph-mgr-rook`
```

Comment:
just one thing. pacific's version number is v16.x.y. so probably we should change `>=17.0.0` to `>=16.2.x`?

Reply:
aah, yes! updated to >=16.0.0 thanks!